packages/react-ui-core/src/DropDownMenu/ItemWrapper.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import cn from 'classnames'
+import themed from 'react-themed'
+
+@themed('DropDownMenu', { pure: true })
```

Comment:
I think you need to use `@themed(/^DrownDownMenu/, { pure: true })` since you're using `DropDownMenu_ListItem` and `DropDownMenu_Selected`.